src/components/HomePage/SearchArea/useUrlSearchParams.js

```diff
@@ -75,24 +79,33 @@ export default ({
         if (!showJobDurationSlider)
             changeURLFilters(location, queryParams, { jobMinDuration: null, jobMaxDuration: null });
 
+        /* istanbul ignore else */
         if (setShowJobDurationSlider)
             setShowJobDurationSlider(showJobDurationSlider);
 
     }, [changeURLFilters, location, queryParams, setShowJobDurationSlider]);
 
     const actualSetFields = useCallback((fields) => {
 
+        // eslint-disable-next-line no-param-reassign
+        fields = ensureArray(fields);
```

Comment:
Why not just create a new variable? The eslint error probably exists for a valid reason